crates/nu-command/tests/commands/mut_.rs

```diff
@@ -171,3 +171,15 @@ fn mut_path_operator_assign() {
 
     assert_eq!(actual.out, "5");
 }
+
+#[test]
+fn mut_records_update_properly() {
+    let actual = nu!(
+        cwd: ".", pipeline(
+        r#"
+        mut a = {}; $a.b.c = 100; $a.b.c
+        "#
+    ));
```

Comment:
oh just one thing, sorry...

i think we do not need `cwd` nor `r#` here :wink: